solver/src/solver/paraswap_solver.rs

```diff
@@ -57,40 +60,132 @@ impl ParaswapSolver {
     }
 }
 
+fn is_retryable_err(err: &SettlementError) -> bool {
+    match err {
+        SettlementError::Paraswap(ParaswapResponseError::PriceChange) => {
+            tracing::warn!("Paraswap Price Change");
+            true
+        }
+        SettlementError::Paraswap(ParaswapResponseError::BuildingTransaction(query)) => {
```

Comment:
Could you point me to the discussion that says this is something we should be retrying? From https://gnosisinc.slack.com/archives/C0203Q9R4JJ/p1625060019006500 I took this shouldn't happen anymore?

Reply:
Yes, so I have not managed to uncover any text regarding this being something we should retry. I am beginning to suspect this was a discussion I had with @nlordell somewhere that isn't visible (or written) so I will try and reconstruct the reason for the decision. 

In the thread you linked to it was concluded that they had halved the ping time from 2 seconds to 1 second and this was supposed to resolve the issue. However these errors are still occurring (about once per day). Because this was a ping related issue, we thought that retrying could be a good way to circumvent this.